crates/ruff_linter/src/rules/pydocstyle/rules/indent.rs

```diff
@@ -287,10 +316,13 @@ pub(crate) fn indent(checker: &mut Checker, docstring: &Docstring) {
         }
 
         // If the last line is over-indented...
-        if let Some(last) = lines.last() {
-            let line_indent = leading_space(last);
+        if let Some(last) = last {
+            let line_indent = leading_space(&last);
             let line_indent_size = line_indent.chars().count();
-            if line_indent_size > docstring_indent_size {
+            let last_line_over_indent = line_indent_size.saturating_sub(docstring_indent_size);
+
+            let is_indent_only = line_indent.len() == last.len();
+            if last_line_over_indent > 0 && is_indent_only {
```

Comment:
This requires the same logic. Only apply the last line logic when it is the last line and it only contains the quotes